### PR TITLE
Have skeleton of file set up with webpack, dependencies, and linter

### PR DESCRIPTION
**Pull request by ZeMunchkin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  },
+  "homepage": "https://github.com/FoodiGo/Reviews#readme",
+  "dependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "body-parser": "^1.18.2",
+    "express": "^4.16.2",
+    "install": "^0.10.4",
```
Comment (MichaelDurfey) [inline package.json]: What's install? Just want to make sure this wasn't a typo here.

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets" : ["es2015", "react"]
```
Comment (MichaelDurfey) [inline .babelrc]: Consider using the env preset here because es2015 has been deprecated.

Comment (ZeMunchkin) [reply .babelrc]: Thanks for this tip. Fixed!

File: package.json (hunk 1 above)
Comment (ZeMunchkin) [reply package.json]: Nice catch

**Pull request merged**